src/components/record/Record.jsx

```diff
@@ -25,6 +28,10 @@ class Record extends React.Component {
     this.recordForm = React.createRef();
     this.state = {
       isFormValid: false,
+      form: null,
+      showModal: false,
+      invalidQuestions: [],
+      incompleteQuestions: [],
```

Comment:
Not sure why we need both at the same time but I guess it is ok :)

Reply:
Tbh the main reason is that otherwise, I had no idea how to name it if it merges both

Reply:
- invalidQuestions = required to save OR not empty answers invalid to save
- incompleteQuestions = required to complete

Reply:
ok